common/scripts/setup_env.sh

```diff
@@ -175,17 +175,14 @@ fi
 # LOCAL_OUT should point to architecture where we are currently running versus the desired.
 # This is used when we need to run a build artifact during tests or later as part of another
 # target.
-if [[ "${FOR_BUILD_CONTAINER:-0}" -eq "1" ]]; then
-  LOCAL_OUT="${TARGET_OUT_LINUX}"
-else
-  LOCAL_OUT="${TARGET_OUT}"
-fi
-
 if [[ "${FOR_BUILD_CONTAINER:-0}" -eq "1" ]]; then
   # Override variables with container specific
   TARGET_OUT=${CONTAINER_TARGET_OUT}
   TARGET_OUT_LINUX=${CONTAINER_TARGET_OUT_LINUX}
-  REPO_ROOT="${WORKDIR}" # /work
+  REPO_ROOT=/work
```

Comment:
This change was introduced by Dario, to make WASM extensions work. Perhaps we should keep it.